src/org/loklak/api/search/GithubProfileScraper.java

```diff
@@ -66,53 +70,42 @@ public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
 	public JSONObject serviceImpl(Query call, HttpServletResponse response, Authorization rights, JSONObjectWithDefault permissions)
 			throws APIException {
 		String profile = call.get("profile", "");
+		String termsS = call.get("terms", "");
```

Comment:
This is a weird name to give and a confusing `sS`. Is there an issue if we call it `terms`? That seems better.